### PR TITLE
PS-7740 Release Notes for 8.0.25-15

### PR DESCRIPTION
**Pull request by patrickbirch**
Changes to be committed:
	modified:   source/conf.py
	modified:   source/index.rst
	new file:   source/release-notes/Percona-Server-8.0.25-15.1.rst
	modified:   source/release-notes/release-notes_index.rst

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-8.0.25-15.1.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+:Date: June 16, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.25-15.1
+includes all the features and bug fixes available in
+`MySQL 8.0.25 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-25.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+The TokuDB Storage Engine will be disabled in future Percona Server for MySQL 8.0 versions. We recommend migrating to the MyRocks Storage Engine.
```
Comment (LenzGr) [inline doc/source/release-notes/Percona-Server-8.0.25-15.1.rst]: Please include a pointer to the blog post that provides additional details.
See [PS-7426](https://jira.percona.com/browse/PS-7426) for details.

Comment (LenzGr) [reply doc/source/release-notes/Percona-Server-8.0.25-15.1.rst]: Also, can we make this paragraph stand out using a note box?

File: doc/source/release-notes/Percona-Server-8.0.25-15.1.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+================================================================================
+*Percona Server for MySQL* 8.0.25-15.1
+================================================================================
+
+:Date: June 16, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.25-15.1
+includes all the features and bug fixes available in
+`MySQL 8.0.25 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-25.html>`_
```
Comment (LenzGr) [inline doc/source/release-notes/Percona-Server-8.0.25-15.1.rst:14]: It also contains all changes available in MySQL 8.0.24! These should be included as well.

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.25-15.1.rst:14]: Added.

File: doc/source/release-notes/Percona-Server-8.0.25-15.1.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.25-15.1.rst]: Added the link to the blog post. 
Note added.

File: doc/source/release-notes/Percona-Server-8.0.25-15.1.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+includes all the features and bug fixes available in the
+`MySQL 8.0.24 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-24.html>`__ and
+the
+`MySQL 8.0.25 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-25.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+.. note::
+
+
+    The TokuDB Storage Engine will be disabled in future Percona Server for MySQL 8.0 versions. We recommend migrating to the MyRocks Storage Engine.
```
Comment (LenzGr) [inline doc/source/release-notes/Percona-Server-8.0.25-15.1.rst]: Could you please also include a reminder that we already marked TokuDB as deprecated a while ago?
I provided a suggestion for this in https://jira.percona.com/browse/PS-7426

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.25-15.1.rst]: Added.

**Review by LenzGr — APPROVED** [review]
LGTM, thank you!

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-8.0.25-15.1.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+* :jirabug:`PS-7695`: The ``Boost`` download is no longer available. Using the -DDOWNLOAD_BOOST option with CMake (Thanks to user Benjamin Kuen for reporting this issue).
+* :jirabug:`PS-6802`: Configure fails with make-4.3 with CMake Error at storage/rocksdb/CMakeLists.txt:152 (STRING) (Thanks to user Thomas Deutschmann for reporting this issue).
+* :jirabug:`PS-7648`: Optimizer switch "favor_range_scan" is not documented
+* :jirabug:`PS-7595`: Same version upgrade from PS->PXC needs mysql_upgrade
+* :jirabug:`PS-7657`: A server exit caused with an update query on a partition table with a compressed column
+* :jirabug:`PS-7557`: Mysql server version 8.0.22-13 executing the Data Masking plugin causes a server exit (Thanks to user Alfonso Luciano for reporting this issue).
+* :jirabug:`PS-7526`: Unexpected quoting and dropping of comments in ``DROP TABLE`` commands (Upstream :mysqlbug:`87852`)
+* :jirabug:`PS-1116`: LP #1719506: Audit plugin reports "command_class=error" for server-side prepared statements
+* :jirabug:`PS-7789`: `procs_files_spec` can be edited with SET PERSIST_ONLY.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-8.0.25-15.1.rst]: This is probably not needed as it was not present in previous release and fixed in this one :)

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.25-15.1.rst]: Removed.

**Pull request merged**